Bottom: 08402a096e4b549817d8129fbc071970ba6f58f2
Top:    a711dcaac86b297177eb35cf823ee6637a2a20f2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-27 04:55:04 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 41cadde..3e04d17 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -991,8 +991,8 @@ set_pv_inlayer_color (PinType *pv, LayerType *layer, int type)
   else if (TEST_FLAG (FOUNDFLAG, pv))     gui->graphics->set_color (Output.fgGC, PCB->FoundColor);
   else
     {
-      int top_group = GetLayerGroupNumberByNumber (component_silk_layer);
-      int bottom_group    = GetLayerGroupNumberByNumber (solder_silk_layer);
+      int top_group = GetLayerGroupNumberBySide (TOP_SIDE);
+      int bottom_group = GetLayerGroupNumberBySide (BOTTOM_SIDE);
       int this_group      = GetLayerGroupNumberByPointer (layer);
 
       if (this_group == top_group || this_group == bottom_group)
